rails/benchmarks/bm_activerecord_create.rb

```diff
@@ -4,10 +4,7 @@
 
 require_relative 'support/benchmark_rails.rb'
 
-ActiveRecord::Base.establish_connection(
-  adapter: 'sqlite3',
-  database: ':memory:'
-)
+ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'])
```

Comment:
@kirs I tried out the `DATABASE_URL` for sqlite3 and postgresql.

Works fine on postgresql but we need to pass `sqlite3:/:memory` (note the trailing slash) for Rails 3.2+. Other than that this looks good to me. 

@rafaelfranca Thanks for popping by here and reviewing :smile: